app/(service)/my-store/edit/_components/myStoreForm.tsx

```diff
@@ -69,11 +83,26 @@ export default function MyStoreForm() {
     }
   };
 
-  const onSubmit = () => {
-    // TODO: submit함수 구현
+  const onSubmit = async (formData: any) => {
+    try {
+      const imageUrl = await requestUploadImg(formData.imageUrl[0]);
+      const postData: PostCreateStoreBody = {
+        ...formData,
+        imageUrl,
+        originalHourlyPay: Number(formData.originalHourlyPay),
+      };
+      const { item } = await postCreateStore(postData);
+      alert('내가게가 등록 되었습니다');
+      router.push(`/my-store/${item.id}`);
+    } catch (error) {
+      if (error instanceof AxiosError) {
+        const errorMessage = error.response?.data.message;
+        alert(errorMessage);
+      }
+    }
   };
   return (
-    <form onSubmit={handleSubmit((data) => console.log(data))}>
+    <form onSubmit={handleSubmit((data) => onSubmit(data))}>
```

Comment:
P4: 이렇게만 적어주셔도 괜찮을 것 같습니다.

```suggestion
    <form onSubmit={handleSubmit(onSubmit)}>
```